projects/recipe-page/phase-3-bootstrap-prompt.md

```diff
@@ -38,7 +38,7 @@ Phase 4 of the project involves incorporating jQuery. You'll add at least one in
 ## Setup Instructions
 Use the Terminal for navigating around the file system and creating new folders and files. Refer to the lesson on [The Command Line Interface](https://github.com/Techtonica/curriculum/blob/master/command-line/command-line-interface.md) if you need guidance on using the Terminal.
```

Comment:
Make this link relative too.

Reply:
Make the link in line 45 relative too.

Reply:
I don't think this is in-scope, Kundan - this issue is just about git directory links.